tests/integration/test_application_variables.py

```diff
@@ -60,7 +60,7 @@ def test_google_analytics_data_layer_is_set_to_nisra_false(self):
         self.assertInHead("gtm.start")
         # pylint: disable=line-too-long
         self.assertInHead(
-            f'dataLayer = [{{"nisra": false}}, {{"form_type": "I", "survey_id": "0", "title": "Census individual test schema"}}, {{"tx_id": "{actual["METADATA"]["tx_id"]}"}}]'
+            f'dataLayer = [{{"form_type": "I", "survey_id": "0", "title": "Census individual test schema"}}, {{"tx_id": "{actual["METADATA"]["tx_id"]}"}}]'
```

Comment:
Title is census individual test schema, should this still be here?

Reply:
This test itself can be removed. It doesn't do anything different anymore.

Reply:
Pete and I recreated it with another schema and the test passes - I'm not sure if this was required and if it should still be deleted?

Reply:
It no longer tests anything different to `test_google_analytics_data_layer_has_no_null_fields`.

Reply:
No problem, deleted 👍